### PR TITLE
add Tips & tricks section to admin settings

### PR DESCRIPTION
**Pull request by jancborchardt**
I still need to know how to properly link the URLs in the documentation. Using `<?php p(link_to_docs('admin_manual')); ?>` didn’t really work, as you can see.

Please review @karlitschek @DeepDiver1975 @danimo 

Also, pages regarding »How to secure your ownCloud« and »How to scale your ownCloud« would be cool, but have to be written first. cc @LukasReschke

### CODE REVIEW

**Comment by jancborchardt** [comment]
Also @DeepDiver1975 how can we make a link to the »Tips & tricks« section show up in the sidebar as well?

**Comment by DeepDiver1975** [comment]
> Also @DeepDiver1975 how can we make a link to the »Tips & tricks« section show up in the sidebar as well?

fixed

**Comment by DeepDiver1975** [comment]
> I still need to know how to properly link the URLs in the documentation. Using <?php p(link_to_docs('admin_manual')); ?> didn’t really work, as you can see.

the keys have to be defined in go.php in the docs repo - https://github.com/owncloud/documentation/blob/master/go.php#L7

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by jancborchardt** [comment]
Added the correct redirects, and put the keys in the go.php in the documentation: https://github.com/owncloud/documentation/pull/980

Please review @DeepDiver1975 @nickvergessen

**Comment by jancborchardt** [comment]
@owncloud-bot retest this please

File: settings/templates/admin.php [hunk 1]
```diff
@@ -111,7 +111,7 @@
 		</strong>
 	</p>
 	<p>
-		<?php p($l->t('To migrate to another database use the command line tool: \'occ db:convert-type\'')); ?>
+		<?php print_unescaped($l->t('To migrate to another database use the command line tool: \'occ db:convert-type\', or see the <a target="_blank" href="%s">documentation ↗</a>.', "https://doc.owncloud.org/server/8.0/admin_manual/configuration_database/db_conversion.html?highlight=sqlite")); ?>
```
Comment (nickvergessen) [inline settings/templates/admin.php]: Should this be go.php?

Comment (jancborchardt) [inline settings/templates/admin.php]: Good catch – fixed.

**Pull request merged**